README.md

```diff
@@ -70,6 +70,16 @@ Currently, zk-Harness includes the following components as a starting point:
 
 We aim to successively expand this list to further include benchmarks for other ZKP frameworks, recursion and zk-EVMs. As a part of the ZKP/Web3 Hackathon hosted by UC Berkeley RDI, we aim to further develop the frameworks integrated into zk-Harness. You can find the program description detailing future integrations [here](https://drive.google.com/file/d/1Igm47dFXSOFAC_wldfUG4Y9OiITqlbQu/view). A detailed list of currently included sub-components and the framework architecture can be found in the [GitHub](https://github.com/zkCollective/zk-Harness) repository.
 
+## How to use
```

Comment:
Please add two subsections: one for running without nix (following installation instructions in each sub-folder), and another one using nix.

Reply:
Hi @StefanosChaliasos , I'll have to leave it to you to instruct on how to use these benchmarks without Nix, because I don't know how to.

Reply:
I think it's also worth asking, why bother supporting both Nix and non-Nix usage? If you're going to use Nix, then you get the benefit (in theory) that everyone can set up a functionally equivalent environment for running the benchmarks, with a minimum of hassle. This reduces the burden of troubleshooting problems which are of the nature of "it doesn't behave the same on your machine as it does on mine." If you support both Nix and non-Nix environments, then you get the costs of supporting Nix (writing Nix code, making shell scripts runnable in a non-FHS-compliant Nix environment, etc.), but you don't get the benefit of everybody running the code in similar environments, and you do get the costs of supporting non-Nix environments. It's sort of a worst of both worlds scenario. If it was my project, then I would support Nix usage and I would refrain from supporting non-Nix usage, or if non-Nix usage was a must have for some reason, then I would just support that non-Nix usage. For a project like this, which has a wide dependency footprint, using Nix seems like a pretty big win as soon as you want multiple people to run these benchmarks, but that's just my humble opinion.

Reply:
I agree that it is the best option to run it through Nix, but some users may want to run a specific benchmark quickly without having to install Nix and all dependencies. I will update the README outside of this PR.